Controllers/StudentController.js

```diff
@@ -0,0 +1,129 @@
+const AsyncHandler = require("express-async-handler");
+const bcrypt = require("bcrypt");
+const jwt = require("jsonwebtoken");
+const Student = require("../Models/StudentModel");
```

Comment:
split npm packages with your packages. Kindly make this change every where in these files